src/classes/CRLP_RollupContact_TEST.cls

```diff
@@ -543,4 +543,85 @@ public class CRLP_RollupContact_TEST {
             System.assertEquals(null, c3.npo02__NumberOfClosedOpps__c);
         }
     }
+
+     /**
+     * @description Multi-Currency Specific tests:
+     * 1. Verifies the SUM on numeric source field will not use currency conversion when calculating the target field value
+     * 2. Verifies that a SingleResultRollup (Largest Gift) is properly converted
+     */
+    @isTest
+    private static void shouldHandleCurrencyConversionWhereNeeded() {
+        final Decimal oppAmountUSD = 100;
+        final Decimal oppAmountCAD = 500;
+
+        if (!UserInfo.isMultiCurrencyOrganization()) {
```

Comment:
@force2b , can we split this test into two since the one testing the numeric value should be executed regardless if the org is multi-currency org. It seems we are verifying more than one concept in this test. Thoughts?

Reply:
I decided to refactor the test to do both operations to avoid the performance impact of running another CRLP test, but I also realized that it needed to only run for multi-currency. The purpose of the original test method is to validate that the sum works properly in a multi-currency org. We're already testing sums in the main test method so there isn't a need to re-test again here. This method really should be for multi-currency only. Otherwise it's not testing anything that isn't already tested. That noted, it would make sense to split into two tests because neither would run in a single-currency org.